pkg/oci/block/block.go

```diff
@@ -30,50 +31,59 @@ const (
 	// FIXME: Assume lun 1 for now?? Can we get the LUN via the API?
 	diskIDByPathTemplate = "/dev/disk/by-path/ip-%s:%d-iscsi-%s-lun-1"
 	volumeOCIDTemplate   = "ocid1.volume.oc1.%s.%s"
+	ocidVolPrefix        = "ocid1.volume."
+	ocidPrefix           = "ocid1."
 )
 
 // OCIFlexvolumeDriver implements the flexvolume.Driver interface for OCI.
 type OCIFlexvolumeDriver struct{}
```

Comment:
`block.OCIFlexvolumeDriver -> block.FlexvolumeDriver` maybe now w/ the new pkg structure?